fixtures/host-2376rb.api.swiftype.com-443/153114567899185607

```diff
@@ -0,0 +1,28 @@
+POST /api/as/v1/engines/node-modules/search.json
+authorization: Bearer api-hean6g8dmxnm2shqqiag757a
+content-type: application/json
+x-swiftype-client: swiftype-app-search-javascript
```

Comment:
This is what was added:

```
x-swiftype-client: swiftype-app-search-javascript
x-swiftype-client-version: 1.2.1
```

There are dynamically pulled from package.json, so they should remain perpetually up to date.

They are implicitly tested here, because if these are not included on an outbound request, the request will fail. The specs now expect these headers to be in place.